docker-compose-localdev.yml

```diff
@@ -55,10 +55,29 @@ services:
       timeout: 5s
       retries: 5
 
+  explorationmodule_postgis:
+    image: "postgis/postgis:15-3.3-alpine"
+    restart: on-failure
+    environment:
+      - POSTGRES_USER=edpn_explorationmodule
+      - POSTGRES_PASSWORD=TvhRZP7b6Hrw64Ct
+      - POSTGRES_DB=explorationmodule
+    volumes:
+      - "explorationmodule_postgis:/var/lib/postgresql/data"
+    ports:
+      - "54322:5432"
+    healthcheck:
+      test: pg_isready
```

Comment:
this health check will likely fail on hosted environments, will it not?

Reply:
that is because this compose should not be used in a hosted environment,

this is the container setup to be able to debug the code locally